tests/phpstan/configs/actual-problems.neon

```diff
@@ -580,6 +580,11 @@ parameters:
 			count: 1
 			path: ../../../src/network/mcpe/ChunkRequestTask.php
 
+		-
+			message: "#^Match expression does not handle remaining values\\: pocketmine\\\\crafting\\\\FurnaceType\\:\\:CAMPFIRE\\|pocketmine\\\\crafting\\\\FurnaceType\\:\\:SOUL_CAMPFIRE$#"
+			count: 1
+			path: ../../../src/network/mcpe/InventoryManager.php
+
```

Comment:
This is not the way. A `LogicException` or `AssumptionFailedError` in the undesired cases would be more appropriate.
I've fixed it but please don't do this again.